Parsers/ZPA/ZPAEvent.txt

```diff
@@ -0,0 +1,292 @@
+            ModifiedBy=column_ifexists('ModifiedBy', ''),
+            RequestID=column_ifexists('RequestID', ''),
+            AuditOldValue=column_ifexists('AuditOldValue', ''),
+            AuditNewValue=column_ifexists('AuditNewValue', ''),
+            AuditOperationType=column_ifexists('AuditOperationType', ''),
+            ObjectType=column_ifexists('ObjectType', ''),
+            ObjectName=column_ifexists('ObjectName', ''),
+            ObjectID=column_ifexists('ObjectID', ''),
+            CustomerID=column_ifexists('CustomerID', ''),
+            Exporter=column_ifexists('Exporter', ''),
+            TimestampRequestReceiveStart=column_ifexists('TimestampRequestReceiveStart', ''),
+            TimestampRequestReceiveHeaderFinish=column_ifexists('TimestampRequestReceiveHeaderFinish', ''),
+            TimestampRequestReceiveFinish=column_ifexists('TimestampRequestReceiveFinish', ''),
+            TimestampRequestTransmitStart=column_ifexists('TimestampRequestTransmitStart', ''),
+            TimestampRequestTransmitFinish=column_ifexists('TimestampRequestTransmitFinish', ''),
+            TimestampResponseReceiveStart=column_ifexists('TimestampResponseReceiveStart', ''),
+            TimestampResponseReceiveFinish=column_ifexists('TimestampResponseReceiveFinish', ''),
+            TimestampResponseTransmitStart=column_ifexists('TimestampResponseTransmitStart', ''),
+            TimestampResponseTransmitFinish=column_ifexists('TimestampResponseTransmitFinish', ''),
+            TotalTimeRequestReceive=column_ifexists('TotalTimeRequestReceive', ''),
+            TotalTimeRequestTransmit=column_ifexists('TotalTimeRequestTransmit', ''),
+            TotalTimeResponseReceive=column_ifexists('TotalTimeResponseReceive', ''),
+            TotalTimeResponseTransmit=column_ifexists('TotalTimeResponseTransmit', ''),
+            TotalTimeConnectionSetup=column_ifexists('TotalTimeConnectionSetup', ''),
+            TotalTimeServerResponse=column_ifexists('TotalTimeServerResponse', ''),
+            HttpRequestMethod=column_ifexists('Method', ''),
+            NetworkApplicationProtocol=column_ifexists('Protocol', ''),
+            UrlHostname=column_ifexists('Host', ''),
+            UrlOriginal=column_ifexists('URL', ''),
+            HttpUserAgentOriginal=column_ifexists('UserAgent', ''),
+            HttpRequestXff=column_ifexists('XFF', ''),
+            NameID=column_ifexists('NameID', ''),
+            HttpStatusCode=column_ifexists('StatusCode', ''),
+            HttpRequestBodyBytes=column_ifexists('RequestSize', ''),
+            HttpResponseBodyBytes=column_ifexists('ResponseSize', ''),
+            SrcPortNumber=column_ifexists('ClientPublicPort', ''),
+            EventResultDetails=column_ifexists('ConnectionReason', ''),
+            CorsToken=column_ifexists('CorsToken', ''),
+            Origin=column_ifexists('Origin', '')
+       | project
+            LogTimestamp,
+            Customer,
+            SessionID,
```

Comment:
Does this should be "NetworkSessionId". Please refer the link for naming convection https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema

Reply:
updated